groups/groups.yaml

```diff
@@ -153,3 +153,4 @@ groups:
       - xandergrzyw@gmail.com # CoCC
       - xingyang105@gmail.com
       - ynli@google.com
+      - wg-serving-leads@kubernetes.io # WG Serving leads
```

Comment:
So far we have not been doing this. So can we please not add this line here?

Reply:
so I should include people one by one? Is there any troubles with group memberships? I saw it being used in some other groups

Reply:
Also this list already have an example: 

https://github.com/kubernetes/k8s.io/blob/5c1916e60425471b3889daf9b34518943c02c277/groups/groups.yaml#L51

Reply:
I'll clean that up @SergeyKanzhelev. must have missed that

Reply:
fixed. Just for my education, what is wrong with aliases in this list?

Reply:
Hi @dims , can we move forward with this PR?

Reply:
let's please drop `wg-serving-leads@kubernetes.io` from this list.

Reply:
Now actually fixed it. My excuse it that I trusted VSCode UI for PR editing, but it left the change unmerged. Sorry about it